src/components/SidebarContents/SidebarContents.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { scrollWithOffset } from '../TableOfContents';
+import * as Style from './SidebarContents.style';
+
+export interface Item {
+  id: string;
+  title: string;
+}
+
+const SidebarContents: React.FC<{ items: Item[] }> = ({ items }) => {
+  return (
+    <nav>
+      <Style.ItemList>
+        {items.map(({ title, id }) => (
+          <li key={id}>
+            <Style.NavLink
+              to={`#${id}`}
+              scroll={el => scrollWithOffset(el, -80)}
```

Comment:
for later: we should maybe put this `-80` as a const somewhere since its used in a bunch of instances of scrollWithOffset